docs/src/hugo/layouts/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{ partial "head.html" . }}
+
+  <body>
+    <!-- navigation -->
+    {{ partial "top_nav.html" . }}
+    <!-- e/o navigation -->
+
+    <main role="main" class="home">
+
+      <div class="jumbotron jumbosplash bg-white">
+        <div class="container">
+          <!-- Example row of columns -->
+          <div class="row splash">
+            <div class="col-md-8 offset-md-2">
+              <div class="spash-logo">
+                <img src="/img/nelson-with-background-and-text.svg" height="140px" />
+                <h2 class="tagline">Automated, multi-region container deployment</h2>
+                <p>
+                  <a class="btn btn-primary btn-lg" href="/introduction/" role="button">Getting Started</a>
+                  &nbsp;
+                  <a class="btn btn-light btn-lg"
+                     href="https://hub.docker.com/r/getnelson/nelson/tags/"
+                     target="_blank"
+                     role="button">Download
+                  {{ with .Site.Data.global }}
+                    {{ .release_version }}
+                  {{ else }}
+                    {{ $.Site.Params.version }}
+                  {{ end }}</a>
+                </p>
+              </div>
+
+            </div>
+          </div>
+        </div>
+      </div> <!-- /jumbotron -->
+
+      <div class="jumbotron jumbosplash bg-blue-5">
+        <div class="container">
+          <div class="row">
+            <div class="col-md-12">
+              <h2 class="semibold">Open-source, Container-first Orchestration</h2>
+              <p class="lead">
+                Nelson is a cutting edge orchestration engine that empowers users to focus on building their products, whilst providing operators with the safety and confidence user applications are meeting the organizationl best practices. Having its roots in type-theory and academic research, Nelson fuses field experience with formal rigor to make deployments something that "just works".
```

Comment:
As much as I am amused at the mentions, I feel like putting "type theory" and "academic research" might scare some people off, at least on the splash page. Maybe we can put something vaguer or more general, or just even drop the first chunk and start with "Nelson fuses field experience with formal rigor.."

Also maybe replace "users" which can be vague (like end-users?) with "service owners" or "product owners."